crates/chia-datalayer/src/lib.rs

```diff
@@ -1,42 +1,155 @@
-use std::collections::HashMap;
+// use std::collections::HashMap;
 
-type Index = usize;
-type Key = Vec<u8>;
+type TreeIndex = u32;
+// type Key = Vec<u8>;
+type Hash = [u8; 32];
```

Comment:
I don't think this type alias is useful

Reply:
It might end up being a newtype as well, in time.  We'll see.

Reply:
I did make a `TreeHash` type in `clvm-utils` for a similar reason :P